packages/tools/client-debugger/client-debugger-view/src/components/FluidDataView.tsx

```diff
@@ -0,0 +1,60 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import React from "react";
+import { HasContainerId, VisualNode, VisualNodeKind } from "@fluid-tools/client-debugger";
+import { FluidHandleView } from "./FluidHandleView";
+import { TreeView } from "./TreeView";
+import { ValueView } from "./ValueView";
+import { UnknownDataView } from "./UnknownDataView";
+
+/**
+ * {@link FluidDataView} input props
+ */
+export interface FluidDataViewProps extends HasContainerId {
+	containerId: string;
+	node: VisualNode;
+}
+
+/**
+ * Displays visual summary trees for DDS_s within the container
+ */
+export function FluidDataView(props: FluidDataViewProps): React.ReactElement {
+	const { containerId, node } = props;
+
+	let view: React.ReactElement;
+	switch (node.nodeKind) {
+		/**
+		 * node with children
+		 * TreeNodeBase
+		 */
+		case VisualNodeKind.TreeNode:
```

Comment:
We may need separate components for Fluid vs non-Fluid value and tree nodes. Sort of depends on whether or not we need to include the object ID in the visual output. We may also wish to visually distinguish Fluid from non-Fluid nodes in our visual tree via icons or something. Probably not needed for this PR, but something to consider.